### PR TITLE
add adguard to dns section

### PR DESCRIPTION
**Pull request by zerwes**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software) must be added to `non-free.md` and have the license set to `⊘ Proprietary`
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding has working installation instructions.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

awesome-foss/awesome-sysadmin#473

### CODE REVIEW

**Comment by zerwes** [comment]
> > #1825
> 
> Although it was from 2019, the idea may stand to add Adguard to sysadmin

quite confusing ...

**Comment by zerwes** [comment]
> If adgaurd does stay, then the following changes will need to be made.

Thank you for the update

**Comment by nodiscc** [comment]
> Although it was from 2019, the idea may stand to add Adguard to sysadmin
>> quite confusing ...

The recent changes (https://github.com/awesome-foss/awesome-sysadmin/pull/473, https://github.com/awesome-selfhosted/awesome-selfhosted/pull/3938, https://github.com/awesome-selfhosted/awesome-selfhosted/pull/3939, https://github.com/awesome-selfhosted/awesome-selfhosted/pull/3940, https://github.com/awesome-selfhosted/awesome-selfhosted/pull/3941, https://github.com/awesome-selfhosted/awesome-selfhosted/pull/3942, https://github.com/awesome-foss/awesome-sysadmin/pull/504) should clear up any confusion:

[awesome-selfhosted/DNS](https://github.com/awesome-selfhosted/awesome-selfhosted#dns) is now about `DNS servers and management tools with advertisement blocking functionality, primarly aimed at home or small networks`, while [awesome-sysadmin/DNS Servers](https://github.com/awesome-foss/awesome-sysadmin#dns---servers) and [awesome-sysadmin/DNS Control Panels & Domain Management](https://github.com/awesome-foss/awesome-sysadmin#dns---control-panels--domain-management) are about `DNS servers, control panels, web interfaces and domain management tools.`

We could rework this a bit more, but at least it's now clear that DNS-based ad-blocking solutions (whic AdGuard is) go to awesome-selfhosted.

File: README.md [hunk 1]
```diff
@@ -753,6 +753,7 @@ _See also: [dbdb.io - Database of Databases](https://dbdb.io/)_
 
 _See also: [awesome-sysadmin/DNS - Servers](https://github.com/awesome-foss/awesome-sysadmin#dns---servers), [awesome-sysadmin/DNS - Control Panels & Domain Management](https://github.com/awesome-foss/awesome-sysadmin#dns---control-panels--domain-management)_
 
+- [AdGuard Home](https://github.com/AdguardTeam/AdGuardHome) - Free and open source, userfriendly ads & trackers blocking DNS server. `GPL-3.0` `Docker`
```
Comment (nodiscc) [inline README.md:756]:
```suggestion
- [AdGuard Home](https://adguard.com/en/adguard-home/overview.html) - Free and open source, userfriendly ads & trackers blocking DNS server. ([Source Code](https://github.com/AdguardTeam/AdGuardHome)) `GPL-3.0` `Docker`
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**